src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString.trim().split(';')
+    .map(item => item.trim().split(':'))
+    .filter(item => item.length > 1)
+    .forEach(([name, property]) => {
+      styles[name.trim()] = property.trim();
```

Comment:
Since we went through the properties in the beginning and removed whitespace with trim(), we no longer need trim() here

`styles[name] = property;`